### ISSUE TITLE
Light/shadow visibility system is broken

### ISSUE THREAD

**Issue by LaoGaoBeBePao** [fholger/thedarkmodvr#29]
Good example is "CoS3: Sacricide".
First dark corner nearby the road (Where the guard is patrolling in white light). In flat TheDarkMod this place is "dark" and guard can't see me. In TheDarkModVR lightgem is fully lit and the guard kick my ass immediately.

In this FM it is no problem with light visualisation. In both versions of the game this place is the same.

Same situation in other FMs.
Starting room in "The Crown of Penitence". Put out a candle and open the door to the second room with a torch. First room will be "dark" in flat TDM and "fully light" in VR.
In this room light visualisation is broken. If I look at the wall that against the door it will be light in the room all around. But no light if I look at the door.

I assume this issue is related to "Incorrect light/shadow culling based on mouse position" problem. But it's not only "based on mouse position". Maybe the problem is more complicated.

Tested with "reset seated position".

**Comment by fholger** [comment]
There is definitely an issue with light/shadow interactions, or more likely the light scissors. I can disable them, and that fixes basically all those problems, but unfortunately it also tanks performance, so it's not an option. I'll keep working on it, but unfortunately this problem does not have a clear fix or solution right now.

**Comment by fholger** [comment]
Ok, I actually discovered that I messed something up, which explains at least the disappearing light/shadows at certain angles. There is in fact a workaround until I fix it: create a file `autoexec_vr.cfg` and put the line `set r_ignore 1` in it. That should get rid of a lot of the lighting issues. However, it may not quite solve the visibility issue, as there's definitely something weird going on with the lightgem.
